radiation/fortran/radlw/radlw_main.F90

```diff
@@ -1070,14 +1070,114 @@ subroutine lwrad                                                  &
 
         if ( lcf1 ) then
 
+#ifdef SERIALIZE
+if (iplon == 3) then
+! file: radlw_main.F lineno: #1050
+call ppser_set_mode(0)
+write(ser_count_str, '(i6.6)') ser_count
+write(*,*) 'iplon = ', iplon
+! file: radlw_main.F lineno: #1053
```

Comment:
You shouldn't need the file generated by ppser in version control -- unless you are making custom modifications to this file?

Reply:
Fixed